content/en/docs/ops/integrations/prometheus/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Prometheus
+description: Information on how to integrate with Prometheus.
+weight: 20
+keywords: [integration,prometheus]
+---
+
+[Prometheus](https://prometheus.io/) is an open source monitoring system and time series database. You can use Prometheus with Istio to record metrics that track the health of Istio and of applications within the service mesh. These metrics can be visualized using tools like [Grafana](/docs/ops/intgrations/grafana/) and [Kiali](/docs/tasks/observability/kiali/).
+
+## Configuration
+
+In an Istio mesh, each component will expose an endpoint that emits metrics. Prometheus works by scraping these endpoints and collecting the results. This is is configured through the [Prometheus configuration file](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) that controls which endpoints to query, the port and path to query, TLS settings, and more.
+
+To gather metrics for the entire mesh, Prometheus should be configured to scrape
+
+1. The control plane (`istiod` deployment)
+1. Ingress and Egress gateways
+1. The Envoy sidecar
+1. The user application, if they expose Prometheus metrics
+
+To simplify the configuration of metrics, Istio offers two modes of operation.
+
+### Option 1: Customized scraping configurations
+
+The built-in deployment of Prometheus is bundled with all of the needed scraping configuration. To deploy the built-in Prometheus instance, follow the steps in [Customizable Install with Istioctl](/docs/setup/install/istioctl/) to install Istio and pass `--set values.prometheus.enabled=true` during installation.
+
+This built-in deployment of Prometheus is intended for new users quickly getting started. However, it does not offer advanced customization, like persistence or authentication. To utilize an existing Prometheus deployment, copy the scraping configurations in [`prometheus/configmap.yaml`]({{< github_file>}}/manifests/charts/istio-telemetry/prometheus/templates/configmap.yaml).
```

Comment:
This built-in deployment of Prometheus is intended for new users to help them quickly getting started.

However, it does not offer advanced customization, like persistence or authentication and as such not considered production ready.